.github/workflows/publish-release-reusable.yml

```diff
@@ -94,7 +122,7 @@ jobs:
     # and that perturbs the path enough that `python` above would resolve to an older built-in Python version.
     # Additionally, since the refman build scripts expect to find Dafny in its usual Binaries/ folder (not in
     # a platform-specific directory), we build Dafny once here.
-    - name: Build Dafny
+    - name: Re-build Dafny
       run: dotnet build dafny/Source/Dafny.sln
```

Comment:
I think this step is obsolete.

Reply:
I thought so, too, but the reference manual build fails without it. It may be possible to move that build up _before_ running `package.py`, though.

Reply:
Actually, given the comment in the workflow about the path to `python`, I don't really want to move that earlier.

Reply:
Right, I guess that is the new feature matrix's fault. It's regenerated when building the manual.